### PR TITLE
Removing Server Side Cache

### PR DESCRIPTION
**Pull request by Fosol**
Originally wanted to test the ability to quickly add server side caching for some endpoints, but I think it's causing memory issues in the docker container.

### CODE REVIEW

File: backend/api/Areas/Property/Controllers/SearchController.cs [hunk 1]
```diff
@@ -2,7 +2,6 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http.Extensions;
 using Microsoft.AspNetCore.Mvc;
-using NetTopologySuite.IO;
```
Comment (devinleighsmith) [inline backend/api/Areas/Property/Controllers/SearchController.cs:5]: this is unrelated right? I wonder if we could get basic static analysis going for .net at some point.

**Review by devinleighsmith — APPROVED** [review]

**Review by jdtoombs — APPROVED** [review]

**Review by agalbrai — APPROVED** [review]
Did some reading to understand what was happening 👍

**Comment by goemen** [comment]
Looks good to me

**Review by goemen — APPROVED** [review]

File: backend/api/Areas/Property/Controllers/SearchController.cs (hunk 1 above)
Comment (Fosol) [reply backend/api/Areas/Property/Controllers/SearchController.cs:5]: Unrelated, but an unused `using`.  It would be helpful to have more linting on the backend, but I don't have time to address.
